Resources/Prototypes/_NF/Entities/Objects/Consumable/Food/meals.yml

```
@@ -642,3 +642,60 @@
   - type: Tag
     tags:
     - Meat
+
+- type: entity
+  parent: FoodMealBase
+  id: FoodMeatSausage
+  name: cooked sausage
+  description: A simple sausage. Who knows what's in this thing?
+  components:
+  - type: Tag
+    tags:
+    - Cooked
+    - NFHotdog
+  - type: FlavorProfile
+    flavors:
+      - meat
+  - type: Sprite
+    sprite: _NF/Objects/Consumable/Food/meat.rsi
+    state: sausagecooked
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 10
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 6
+        - ReagentId: Flavorol
+          Quantity: 2
+  - type: Construction
+    graph: hotdogstove
+    node: cookeddog
+
+- type: entity
+  parent: FoodMealBase
+  id: FoodMealHotdog
+  name: hotdog
+  description: Hot diggity dog!
+  components:
+  - type: Tag
+    tags:
+    - Cooked
+  - type: FlavorProfile
+    flavors:
+      - meat
```

```suggestion
      - meaty
```
Minor spelling mistake (doesn't give the flavor when eating it)

I would also suggest adding bread, so we can taste the bun as well!
```suggestion
      - meaty
      - bread
```